server/lists/adapters/workstacks.js

```diff
@@ -147,7 +147,7 @@ const decorateContributionsWithStatus = (contributions, currentDate) => {
 };
 
 const bindDisplayElements = fromStaticList => async (stage) => {
-    stage.assignedTeamDisplay = await fromStaticList('S_TEAMS', stage.teamUUID, true);
+    stage.assignedTeamDisplay = await fromStaticList('S_ALL_TEAMS', stage.teamUUID, true);
```

Comment:
Need to make sure that true is removed from these calls.

Reply:
I've addressed this.